### PR TITLE
update demo to use recommended whiteframe class

### PR DESCRIPTION
**Pull request by amitport**

### CODE REVIEW

**Comment by DevVersion** [comment]
Nice, looks good to me.

File: src/components/sidenav/demoBasicUsage/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
   <section layout="row" flex>
 
-    <md-sidenav class="md-sidenav-left md-whiteframe-z2" md-component-id="left" md-is-locked-open="$mdMedia('gt-md')">
+    <md-sidenav class="md-sidenav-left md-whiteframe-4dp" md-component-id="left" md-is-locked-open="$mdMedia('gt-md')">
```
Comment (ThomasBurleson) [inline src/components/sidenav/demoBasicUsage/index.html]: try `md-whiteframe="4"`

Comment (DevVersion) [inline src/components/sidenav/demoBasicUsage/index.html]: Right, even a better idea.

Comment (ThomasBurleson) [inline src/components/sidenav/demoBasicUsage/index.html]: Ah this is for the basic, class-based usage... not the directive usage. So nm.

**Comment by ThomasBurleson** [comment]
Closed with SHA b4b7040538f2c58da50506e9a868cf6cff69d976

**Pull request closed without merging**